lib/dangermattic/plugins/manifest_pr_checker.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Danger
+  # Plugin to check if the Gemfile.lock was updated when changing the Gemfile in a PR.
+  class ManifestPRChecker < Plugin
```

Comment:
I wonder if it wouldn't be worth also providing a helper method for the call sites, like this:
```ruby
def check_all_manifest_lock_updated
  check_gemfile_lock_updated
  check_podfile_lock_updated
  check_swift_package_resolved_updated
end
```
That would make easier to use for projects that are likely to have them all (and if the project don't have one of them, e.g. they don't use SPM, then `check_manifest_lock_updated` will never raise the warning for those ones since `manifest_modified` will always be `false` for those projects not using `Package.swift` anyway)

wdyt?

Reply:
Ah, it's simple enough to just do it 🙂 it's also nice to have just a single entry point.